osx/daily/build.sh

```diff
@@ -0,0 +1,99 @@
+#! /bin/bash
+
+set -xe
+
+VERSION_SUFFIX=daily
+
+PROJECT_DIR=/Users/admin/daily
+QT_DIR=/usr/local/Qt-5.15.2
+OSX_SYSROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk
+OSX_DEPLOYMENT_TARGET=10.12
+
+USER_PASSWORD=$(security find-generic-password -w -s "USER_PASSWORD")
+NOTARIZATION_ACCOUNT=$(security find-generic-password -w -s "NOTARIZATION_ACCOUNT")
+NOTARIZATION_PASSWORD="@keychain:NOTARIZATION_PASSWORD"
+
+cd $PROJECT_DIR/desktop
+git checkout master
+git pull origin master
+git submodule update
+
+cd ..
+
+rm -rf build
+mkdir build
+cd build
+
+export PATH=/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:$PATH
+
+export PATH=$QT_DIR/bin/:$PATH
+export OPENSSL_ROOT_DIR=$(brew --prefix openssl@1.1)
+
+mkdir install
+
+#Build QtKeychain
+mkdir qtkeychain
+cd qtkeychain
+
+cmake \
```

Comment:
not blocking from me but could we try to use the Ninja generator ?

Reply:
but make sure it is installed

Reply:
I would propose to merge it so we have it done and versioned somewhere. Later we can improve it.